samples/net/sockets/can/src/main.c

```diff
@@ -12,8 +12,12 @@ LOG_MODULE_REGISTER(net_socket_can_sample, LOG_LEVEL_DBG);
 #include <net/socket.h>
 #include <net/socket_can.h>
 
+#ifdef SAMPLE_SOCKET_CAN_LOOPBACK_MODE
+#include <drivers/can.h>
+#endif
+
 #define PRIORITY  k_thread_priority_get(k_current_get())
-#define STACKSIZE 1024
+#define STACKSIZE 2048
```

Comment:
just curious but was there a particular driver that caused this stack to have to grow or is this a general result of the split out?

Reply:
I had a kernel panic when I tested on HW. Not sure anymore if it was the stack.